platform/ui/src/elements/form/TextInput.css

```diff
@@ -1,7 +1,37 @@
-.input-ohif {
-  background-color: #b6b6b6;
-  border-color: #b6b6b6;
-  font-family: Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue',
-    sans-serif;
+.input-ohif-container {
   font-size: 1em;
+  color: var(--text-primary-color);
+  font-weight: 400;
 }
+
+.input-ohif-container .input-ohif {
+  display: inline-block;
+  height: 40px;
+  margin: 0 10px 20px;
+  padding: 0 20px;
+  cursor: pointer;
+  border: none;
+  background-color: var(--input-background-color);
+  color: var(--input-placeholder-color);
+  font-size: 10pt;
+  font-weight: normal;
+  width: auto !important;
```

Comment:
Why important?

Reply:
Yeah, old stuff again, I just removed and tested, seems that it's not that important anymore.